StantUiIosLibrary/Classes/Dialogs/DialogView/DialogViewController.swift

```diff
@@ -144,5 +144,35 @@ public class DialogViewController: UIViewController {
                          padding: UIEdgeInsets(top: 0, left: leadingAnchorValue, bottom: 0, right: 0),
                          size: CGSize(width: 0, height: 56))
     }
+    
+    public func updateDialog(title: String, mainView: UIView,
+                             buttons: [DialogButton] = [DialogButton](),
+                             canClickOutside: Bool = true) {
+        self.titleLabel?.text = title
+        
+        self.updateInsideView(withView: mainView)
+        self.update(dialogButtons: buttons)
+        self.updateClicks(canClickOutside)
+    }
+    
+    fileprivate func updateInsideView(withView mainView: UIView) {
+        insideView?.removeFromSuperview()
```

Comment:
What does insideView is? By seeing it I would probably know what it is but just by looking at the variable name it doesn't give me much information about it :/

Reply:
I renamed to updateMainContainerInformation :)